docs/posts/roadmap-2024-H1/index.qmd

```diff
@@ -0,0 +1,383 @@
+that can serve as a standard across the data ecosystem.
+:::
+
+So, Voltron Data supports Ibis for the same reason it supports Apache Arrow.
+Open-source standards make it easier to build the composable data ecosystem and
+reduce friction for data teams looking to swap out components as their needs
+change. If you're already using Ibis as your frontend, Apache Arrow for data
+interchange, and Substrait for intermediary representation, you can swap in
+Theseus with ease!
+
+::: {.callout-important}
+[Ibis is independently governed](https://github.com/ibis-project/governance) and
+not owned by Voltron Data. While currently four out of five members of the
+steering committee are employed by Voltron Data (the fifth being at Alphabet
+working on Google BigQuery), we expect this to change over time as more
+organizations join the project. **Ibis will never be solely available for VoDa
+products.**
+
+Voltron Data also welcomes this dilution of power and influence! A healthy
+open-source project is one that is not controlled by a single entity. This is
+true of [Apache Arrow](https://arrow.apache.org) and other open-source projects
+that Voltron Data employees have been instrumental in building.
+:::
+
+### Who are the core contributors?
+
+The core contributors working full-time on Ibis are employed at Voltron Data,
+with deep experience on successful open-source projects including pandas,
+Apache Arrow, Dask, and more. Everything in the Ibis project is made possible by
+their hard work! They are:
+
+- [**Gil Forsyth**](https://github.com/gforsyth): long-time Ibis contributor and
+ primary maintainer of the `ibis-substrait` package
+- [**Jim Crist-Harif**](https://github.com/jcrist): the engineering manager for
+ the Ibis team at VoDa
+- [**Krisztián Szűcs**](https://github.com/kszucs): long-time Ibis contributor
+ and primary author of the precursor to [the big refactor](#the-big-refactor)
+- [**Naty Clementi**](https://github.com/ncclementi): newest member of the Ibis
+ team at VoDa recently focusing on [geospatial support in
+ DuckDB](#geospatial-improvements)
+- [**Phillip Cloud**](https://github.com/cpcloud): the tech lead for the Ibis
+ team at VoDa
+
+If you're interacting with us on GitHub or Zulip, you'll definitely run into at
+least one of them! They make the Ibis project the delightful software it is
+today and are always happy to help.
+
+### Who else supports Ibis?
+
+Anybody who contributes to Ibis is a supporter of Ibis! You can contribute by
+[opening an issue](https://github.com/ibis-project/ibis/issues), [submitting a
+pull request](https://github.com/ibis-project/ibis/pulls), [using Ibis in your
+project](https://github.com/ibis-project/ibis/network/dependents), or [joining
+the Zulip chat](https://ibis-project.zulipchat.com) to discuss problems or
+ideas.
+
+Notable organizations that support Ibis include:
+
+- [**Claypot AI**](https://www.claypot.ai/): contributing the Apache Flink
```

Comment:
Should be mentioned something about them being part of VoDa now? and maybe mention their names? 
I'm thinking that we will be putting efforts into streaming and ML and they'll most likely lead this efforts.

Reply:
it is mentioned a couple times below -- will re-look at this

Reply:
I'm going to leave it as a historical contributed the Flink backend pre-acquisition